modules/mozillians/main.tf

```diff
@@ -0,0 +1,101 @@
+variable "environment" {}
+variable "vpc_id" {}
+variable "service_security_group_id" {}
+variable "elasticache_instance_size" {}
+variable "elasticache_subnet_group" {}
+variable "elasticsearch_instance_type" {}
+variable "mozillians-es-iam-policy" {
+    type    = "string"
+    default = <<CONFIG
+{
+"Version": "2012-10-17",
+"Statement": [
+    {
+        "Action": "es:*",
+        "Principal": "*",
+        "Effect": "Allow",
+        "Condition": {
+          "IpAddress": {
+            "aws:SourceIp": [
+              "52.0.0.42"
```

Comment:
We don't, It's a placeholder until the prod slaves are up. Unfortunately Terraform's `aws_instance` data source can only match to a single instance so we can't use it to populate this.